docs/designs/conceptual-versioning.md

```diff
@@ -526,10 +509,19 @@ In phase 1, when resolving the `toc_rel` of each file, we still take the nearest
 
 For redirection file, the output file also contains `monikers` information.
 
+#### 5.5 Config migration
+
+To support versioning, config migration tool have to:
+
+1. Generate `monikerRange` config item according to the original group config in the `docfx.json`.
+2. Generate `routes` config item according to the original group config in the `docfx.json`
```

Comment:
and `monikerDefinition` too?

Reply:
`MonikerDefinition` will require auth token, so we should put it in global config.